Sources/PrimerSDK/Classes/Data Models/CardNetwork.swift

```diff
@@ -264,7 +264,7 @@ public enum CardNetwork: String, CaseIterable {
         }
     }
     
-    var directoryServerId: String? {        
+    var directoryServerId: String? {
```

Comment:
would be interesting to send an analytic when the card is not recognized 🤔

Reply:
There's an error logged (both iOS and Android), and the 3DS flow ends.